pyuvsim/simsetup.py

```diff
@@ -771,10 +773,12 @@ def parse_frequency_params(freq_params):
 
         if not sf:
             if ef and bw:
-                freq_params['start_freq'] = freq_params['end_freq'] - freq_params['bandwidth'] + freq_params['channel_width']
+                freq_params['start_freq'] = freq_params['end_freq'] - freq_params['bandwidth'] + freq_params[
+                    'channel_width']
         if not ef:
             if sf and bw:
-                freq_params['end_freq'] = freq_params['start_freq'] + freq_params['bandwidth'] - freq_params['channel_width']
+                freq_params['end_freq'] = freq_params['start_freq'] + freq_params['bandwidth'] - freq_params[
+                    'channel_width']
```

Comment:
same as line-breaking comment

Reply:
Fixed in b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0